docs/tutorial/seo-and-social-sharing-cards-tutorial/index.md

```diff
@@ -4,7 +4,7 @@ title: Search Engine Optimization (SEO) and Social Sharing Cards with Gatsby
 
 Perhaps you've been approached by an SEO _expert_ who can maximize your revenue and page views by following these **Three Simple Tricks**! Relatively few people make the concerted effort to implement SEO in their web app. This tutorial will share some of the ins and outs of SEO and how you can implement common SEO patterns in your Gatsby web app, today. By the end of this post you'll know how to do the following:
 
-- Implement SEO patterns with [react-helmet][react-helmet]
+- Implement SEO patterns with the [Gatsby Head API](/reference/built-in-components/gatsby-head/)
```

Comment:
This whole tutorial should be deleted since we already have https://www.gatsbyjs.com/docs/how-to/adding-common-features/adding-seo-component/

Reply:
The only difference is that this one approaches it from a "social cards" perspective, so I thought not to remove it.

If you feeling strongly about removing it then we should

Reply:
Yes, let's remove it